istioctl/cmd/analyze.go

```diff
@@ -110,6 +110,9 @@ func gatherFiles(args []string) ([]string, error) {
 		if err != nil {
 			return nil, err
 		}
+		if len(paths) == 0 {
+			log.Warnf("%q did not resolve to any files and will be ignored.", a)
+		}
```

Comment:
IMO, we error out if any specified file/path doesn't exist. Warnings are easy to miss. Also, this mirrors kubectl behavior.

Reply:
Sounds good. 

As an aside, and looping in @ozevren : while we're here, should we drop the slightly awkward "accept either a file or glob pattern" behavior and only accept files? In nearly all use cases glob expansion is happening in the shell before this code is ever evaluated, so the only time the glob matters is if the user inputs a quoted glob pattern like `"my_test_dir/*"` instead of `my_test_dir/*`.

I'm inclined to drop the glob support unless there are objections.

Reply:
Is there any k8s precedent of a command that accepts a file list?

I agree that supporting expansion from inside the tool itself (as opposed to shell) may not be necessary (and can always be added later in a non-breaking way).